code/game/machinery/doors/brigdoors.dm

```diff
@@ -40,6 +40,9 @@
 /obj/machinery/door_timer/Initialize()
 	. = ..()
 
+	return INITIALIZE_HINT_LATELOAD

```

Comment:
To make code clear, remove the `. =` bit from the above line, since you have no intention of returning that.

So
```dm
    ..()

    return INITIALIZE_HINT_LATELOAD
```